src/arche/rules/others.py

```diff
@@ -6,43 +6,44 @@
 import pandas as pd
 
 
-def compare_boolean_fields(source_df, target_df):
+def compare_boolean_fields(source_df: pd.DataFrame, target_df: pd.DataFrame):
     source_bool = source_df.select_dtypes(include="bool")
     target_bool = target_df.select_dtypes(include="bool")
 
     result = Result("Boolean Fields")
     if not fields_to_compare(source_bool, target_bool):
-        result.add_info("No fields to compare")
+        result.add_info("SKIPPED")
         return result
 
-    source_relative_fr = get_bool_relative_frequency(source_bool)
-    target_relative_fr = get_bool_relative_frequency(target_bool)
-    relative_diffs = abs(source_relative_fr - target_relative_fr) * 100
+    lefts = []
+    for c in source_bool:
+        lefts.append(source_bool[c].value_counts(normalize=True).add_prefix(f"{c}_"))
```

Comment:
In some cases might be useful to report the NaN values for value counts:

`source_bool[c].value_counts(normalize=True, dropna=False)`

Reply:
nan is for coverage, in this context nan equals False.